### PR TITLE
Buffs bloodbait

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

Adds bog trolls, normal trolls, lesser brown moles and goblins to the bloodbait attraction chance.

## Why It's Good For The Game

Bloodbait seems like the most useless one, now it's the most dangerous one.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by ThePainkiller** [comment]
I'd suggest removing goblins (makes no sense when you can just trigger ambushes is usually the way to get them) because vampires will abuse this to farm vitae, and lowering troll spawn chance a fair bit (they are now dangerous as hell)

**Comment by BadAtThisGame302** [comment]
> I'd suggest removing goblins (makes no sense when you can just trigger ambushes is usually the way to get them) because vampires will abuse this to farm vitae, and lowering troll spawn chance a fair bit (they are now dangerous as hell)

Done.

File: code/game/objects/items/rogueitems/bait.dm [hunk 1]
```diff
@@ -104,4 +104,8 @@
 	desc = "A deployable bag of bait used by hunters to attract predators within the wilds."
 	icon_state = "baitb"
 	attracted_types = list(/mob/living/simple_animal/hostile/retaliate/rogue/wolf = 20,
-						/mob/living/simple_animal/hostile/retaliate/rogue/bigrat = 10)
+						/mob/living/simple_animal/hostile/retaliate/rogue/bigrat = 10,
+						/mob/living/simple_animal/hostile/retaliate/rogue/mole = 10,
+						/mob/living/simple_animal/hostile/retaliate/rogue/troll = 10,
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueitems/bait.dm:109]:
```suggestion
						/mob/living/simple_animal/hostile/retaliate/rogue/troll = 5,
```

File: code/game/objects/items/rogueitems/bait.dm [hunk 2]
```diff
@@ -104,4 +104,8 @@
...
+						/mob/living/simple_animal/hostile/retaliate/rogue/trollbog = 10
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueitems/bait.dm:110]:
```suggestion
						/mob/living/simple_animal/hostile/retaliate/rogue/trollbog = 5
```

File: code/game/objects/items/rogueitems/bait.dm [hunk 3]
```diff
@@ -104,4 +104,8 @@
 	desc = "A deployable bag of bait used by hunters to attract predators within the wilds."
 	icon_state = "baitb"
 	attracted_types = list(/mob/living/simple_animal/hostile/retaliate/rogue/wolf = 20,
-						/mob/living/simple_animal/hostile/retaliate/rogue/bigrat = 10)
+						/mob/living/simple_animal/hostile/retaliate/rogue/bigrat = 10,
```
Comment (ThePainkiller) [inline code/game/objects/items/rogueitems/bait.dm:107]:
```suggestion
						/mob/living/simple_animal/hostile/retaliate/rogue/bigrat = 5,
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/game/objects/items/rogueitems/bait.dm (hunk 1 above)
Comment (BadAtThisGame302) [reply code/game/objects/items/rogueitems/bait.dm]: Done.

File: code/game/objects/items/rogueitems/bait.dm (hunk 3 above)
Comment (BadAtThisGame302) [reply code/game/objects/items/rogueitems/bait.dm]: Done.

File: code/game/objects/items/rogueitems/bait.dm (hunk 2 above)
Comment (BadAtThisGame302) [reply code/game/objects/items/rogueitems/bait.dm]: Done.

**Review by ThePainkiller — APPROVED** [review]
Fuck yeah huntable trolls

**Pull request merged**